src/test/kotlin/io/github/mojira/arisa/modules/RemoveVersionModuleTest.kt

```diff
@@ -0,0 +1,221 @@
+package io.github.mojira.arisa.modules
+
+import io.github.mojira.arisa.utils.RIGHT_NOW
+import io.github.mojira.arisa.utils.mockChangeLogItem
+import io.github.mojira.arisa.utils.mockIssue
+import io.github.mojira.arisa.utils.mockProject
+import io.github.mojira.arisa.utils.mockVersion
+import io.kotest.assertions.arrow.either.shouldBeLeft
+import io.kotest.assertions.arrow.either.shouldBeRight
+import io.kotest.core.spec.style.StringSpec
+
+private val TWO_SECONDS_AGO = RIGHT_NOW.minusSeconds(2)
+private val FIVE_SECONDS_AGO = RIGHT_NOW.minusSeconds(10)
+
+private val VERSION = mockVersion(id = "2", released = true, archived = false)
+
+private val ADD_VERSION = mockChangeLogItem(field = "Version", changedTo = "1")

```

Comment:
These definitions are the cause for your failed tests. Your version that's in affectedVersions has id 2, while the changelog entry suggests a version with id 1 was added.

You should look into why the tests didn't run properly on your machine.